integration_test/third_party_apps_data/applications/active_directory_ds/enable

```diff
@@ -26,5 +26,7 @@ metrics:
         - active_directory_ds
 "
 
+# Stop-Service may fail if the service isn't in a Running state yet.
+(Get-Service google-cloud-ops-agent*).WaitForStatus('Running', '00:03:00')
```

Comment:
how does this work wiith the *? does it call WaitForStatus on each of them?

Reply:
Yeah, it does four calls in total. I experimented without the * to see if the parent service was somehow special and might implicitly block on the child services anyway, but that wasn't the case, so I kept the *.